app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::Order.class_eval do
```

Comment:
The class_eval usage is discouraged in solidus. We'd prefer the prepend pattern.
You can find some inspirations here:

https://guides.solidus.io/customization/customizing-the-backend/#implementing-new-actions
https://github.com/solidusio/solidus_bolt/blob/master/app/decorators/models/solidus_bolt/order_decorator.rb